ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureV2TextBoxCustomization.java

```diff
@@ -12,13 +12,62 @@
  */
 public class ThreeDSecureV2TextBoxCustomization extends ThreeDSecureV2BaseCustomization implements Parcelable {
 
+    private String textFontName;
+    private String textColor;
+    private int textFontSize;
```

Comment:
Whats the point of these 3 variables in all of the customizations? On the `setters` for each, they get set in the `customization` variable, an on the `getters` it returns the value directly from the `customization`. Can we dispose of these?

Reply:
Yeah we tried to remove these but they were needed for the parcelable implementation ☹️

Reply:
Ah good catch! The were needed in the parceling method but not as member variables, just updated to remove those 👍